tests/stubtest_allowlists/py3_common.txt

```diff
@@ -407,9 +407,67 @@ pickle.Unpickler.memo  # undocumented implementation detail, has different type
 re.Pattern.scanner  # Undocumented and not useful. #6405
 tempfile._TemporaryFileWrapper.[\w_]+  # Dynamically specified by __getattr__, and thus don't exist on the class
 
-# Various classes in typing aren't types at runtime. In addition, mypy thinks some special forms are tautologically defined.
-typing.[A-Z]\w+
-typing_extensions\..*
+# Details of runtime definition don't need to be in stubs
+typing_extensions\._SpecialForm
+typing\._SpecialForm
+typing_extensions\.TypeVar
+typing_extensions\.TypeVarTuple
+typing\.Generic
+
+# Special primitives
+typing_extensions\.NamedTuple
+typing_extensions\.LiteralString
+typing_extensions\.Final
+typing_extensions\.Coroutine
+typing_extensions\.Awaitable
+typing_extensions\.AsyncIterable
+typing_extensions\.AsyncGenerator
+typing\.Sized
+typing\.Sequence
+typing\.Reversible
+typing\.Pattern
+typing\.NamedTuple
+typing\.MutableSet
+typing\.MutableSequence
+typing\.MutableMapping
+typing\.Match
+typing\.MappingView
+typing\.LiteralString
+typing\.KeysView
+typing\.Iterator
+typing\.Iterable
+typing\.ItemsView
+typing\.Hashable
+typing\.Coroutine
+typing\.Collection
+typing\.Container
+typing\.ByteString
+typing\.AwaitableGenerator
+typing\.Awaitable
+typing\.AsyncIterator
+typing\.AsyncIterable
+typing\.AsyncGenerator
+typing\.Annotated
+typing\.AbstractSet
+
+# We lie in the stub and say it's read-only
+typing_extensions\.ParamSpec(Args|Kwargs)\.__origin__
+
+# Internal attributes
+.*\.__protocol_attrs__
+.*\.__callable_proto_members_only__
```

Comment:
https://github.com/python/mypy/pull/15294 means we should be able to remove these, come the next mypy release

(No change requested)